web/src/App.vue

```diff
@@ -1,21 +1,42 @@
 <template lang="pug">
 v-app
-  v-toolbar.main-toolbar.darken-4(dark, dense, color="primary")
-    v-toolbar-items
-      v-btn.px-0.square-button.darken-4(flat, color="white", to="/select")
-        v-icon {{ $vuetify.icons.close }}
-    v-spacer
-    v-toolbar-title.title
-      span.font-weight-bold.secondary--text.text--lighten-3 Biomarker Analysis »
-      span.font-weight-bold.white--text  {{ $router.currentRoute.name }}
+  v-toolbar.darken-3(dense, dark, color="primary")
+    v-toolbar-side-icon.logo(@click="$router.push('/')") BA
```

Comment:
This should probably read `VIIME` now that we've settled on the name.

Thoughts on capitalization, @jeffbaumes?

Reply:
Let's do VIIME for now. I've chatted with @jtomeck a bit and it seems there are several routes we could go with capitalization/logo/etc. and I don't think that's the priority right now. We'll do a branding pass later on.